bin/lib/autovoice.js

```diff
@@ -88,10 +89,16 @@ function generatePodcast(rssUrl, voiceId=tts.defaultVoiceId){
 		debug(`generatePodcast: items.length=${items.length}`);
 		const promises = items.map( (item, i) => {
 			item.processingIndex = i;
-			return processItemToMp3(item, voiceId);
+			return processItemToMp3(item, voiceId)
+				.catch(err => {
+					debug(`An error occurred trying to generate speech for item ${item.uuid}. Err:`, err);
+					return null;
+				})
+			;
 		} );
 		return Promise.all(promises);
 	})
+	.then( items => filterNullItems(items) )
```

Comment:
why not inline?
.then( items => { return items.filter( item => {return item != null; ) )